### PR TITLE
Added new HTML Question

### PR DESCRIPTION
**Pull request by MakendranG**
Added new HTML Question Q114 to Q116

### CODE REVIEW

**Comment by MakendranG** [comment]
This is with reference to the issue https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/issues/4949

I have added the below details in this file.

Added new HTML Question Q114 to Q116

Kindly add `hacktoberfest-accepted` label in this PR

File: html/html-quiz.md [hunk 1]
```diff
@@ -1950,3 +1950,42 @@ A disabled fieldset is unusable and un-clickable. [Source](https://www.w3schools
...
 - [x] Specify whether the content of an element should be editable or not
 - [ ] Return the position of the first found occurrence of content inside a string
+
+#### Q114. HTML elements that aren't meant to store content or other elements are called _____ elements.
+
+- [x] void
+- [ ] independent
+- [ ] empty
+- [ ] paragraph
+
```
Comment (artemu78) [inline html/html-quiz.md:1960]:
```suggestion

[Source](https://www.w3.org/TR/2011/WD-html-markup-20110113/syntax.html#syntax-elements)

```

File: html/html-quiz.md [hunk 2]
```diff
@@ -1950,3 +1950,42 @@ A disabled fieldset is unusable and un-clickable. [Source](https://www.w3schools
...
+#### Q115. The _____ attribute of an HTML label element is a referencing mechanism to state what input or element the label is tied to.
+
+- [ ] to
+- [ ] of
+- [ ] with
+- [x] for
+
```
Comment (artemu78) [inline html/html-quiz.md:1967]:
```suggestion
[Source](https://www.w3schools.com/tags/att_label_for.asp)

```

File: html/html-quiz.md [hunk 3]
```diff
@@ -1950,3 +1950,42 @@ A disabled fieldset is unusable and un-clickable. [Source](https://www.w3schools
...
+#### Q117. Which HTML element is the container for all the renderable aspects of the document?
+
+- [ ] head
+- [x] body
+- [ ] title
+- [ ] meta
+
+
+#### Q118. The "h" in the HTML h1 tag is short for _____.
+
```
Comment (artemu78) [inline html/html-quiz.md:1985]:
```suggestion
```

File: html/html-quiz.md [hunk 4]
```diff
@@ -1950,3 +1950,42 @@ A disabled fieldset is unusable and un-clickable. [Source](https://www.w3schools
...
+
+- [ ] head
+- [ ] horizontal
+- [ ] hypertext
+- [x] heading
+
```
Comment (artemu78) [inline html/html-quiz.md:1991]:
```suggestion

[Source](https://www.w3schools.com/html/html_headings.asp)
```

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@MakendranG could you pls review comments, thanks

**Comment by MakendranG** [comment]
@Ebazhanov 

Changes have been reviewed and it is been commited

**Review by Ebazhanov — APPROVED** [review]
LGTM 👍

**Pull request merged**